script.js

```diff
@@ -13,42 +10,67 @@ for (let btn of readMoreBtns) {
 
 // toggle forms
 const contactBtn = document.querySelector('.contact__button');
-const contactForm = document.querySelector('.contact__form');
-
 const orderBtn = document.querySelector('.order__button');
+const contactForm = document.querySelector('.contact__form');
 const orderForm = document.querySelector('.order__form');
 
 contactBtn.addEventListener('click', () => {
   orderForm.classList.remove('active__form');
   contactForm.classList.add('active__form');
 
-  contactBtn.style.borderBottom = '5px solid red';
-  orderBtn.style.borderBottom = 'none';
+  contactBtn.classList.add('active__form-header');
+  orderBtn.classList.remove('active__form-header');
 });
 
 orderBtn.addEventListener('click', () => {
   contactForm.classList.remove('active__form');
   orderForm.classList.add('active__form');
 
-  orderBtn.style.borderBottom = '5px solid red';
-  contactBtn.style.borderBottom = 'none';
+  orderBtn.classList.add('active__form-header');
+  contactBtn.classList.remove('active__form-header');
```

Comment:
this is absolutely fine but we could also probably make it shorter by using toggle

```js
bothBtns.forEach( btn => {
  btn.addEventListener('click', () => {
    contactForm.classList.toggle('form--active');
    orderForm.classList.toggle('form--active');
  })
})
```

That way it would work whenever we click either of the buttons and it's only. In any case, that's not a necessary change, just some refactoring that I think could be done.